internal/naming/names.go

```diff
@@ -158,6 +158,15 @@ const (
 	// PGBouncerLogPath is the pgBouncer default log path configuration
 	PGBouncerLogPath = "/tmp"
 
+	// PGbouncerFullLogPath is the full path to the pgbouncer log file
+	PGBouncerFullLogPath = PGBouncerLogPath + "/pgbouncer.log"
+
+	// PGBouncerPostRotateScript is the script that is run after pgBouncer's log
+	// files have been rotated. The pgbouncer process is sent a sighup signal.
+	// TODO: Should this go here or in controller/postgrescluster/pgbouncer.go?
+	// Or somewhere in the internal/pgbouncer folder, for that matter?
+	PGBouncerPostRotateScript = "pkill -HUP --exact pgbouncer"
```

Comment:
Kinda feel like this should be in the pgbouncer.go

Reply:
Agreed. Moving it.